templates/directory.php

```diff
@@ -192,6 +192,10 @@ function pmpromd_shortcode($atts, $content=null, $code="")
 		else
 			$fields_array = false;
 
+
+		//filter the fields
+		$fields_array = apply_filters( 'pmpro_member_directory_fields', $fields_array, $pu );
```

Comment:
1. This needs phpdoc info
2. `$pu` doesn't seem to be defined anywhere yet
3. Instead of `$pu`, should we be passing the `$shortcode_atts` here? If so, around line 216-233 we could take the `$shortcode_atts` and define it above this so we can pass it in.

```suggestion
		/**
		 * Allow filtering the fields to include on the member directory list.
		 *
		 * @since TBD
		 *
		 * @param array  $fields_array The list of fields to include.
		 */
		$fields_array = apply_filters( 'pmpro_member_directory_fields', $fields_array );
```